pandas/core/strings.py

```diff
@@ -633,21 +633,21 @@ def _str_extract_frame(arr, pat, flags=0):
 
 def str_extract(arr, pat, flags=0, expand=True):
     r"""
+    Return the match object corresponding to regex `pat`.
```

Comment:
Is this `r` before the doc string right?

Reply:
Yes, it is to ensure correct rendering for used `\` in the examples

Reply:
This description isn't quite right, is it? We aren't returning a `Match` object.

Maybe

    Extract capture groups in the regex `pat` as columns in a DataFrame.